core/genesis.go

```diff
@@ -146,9 +146,6 @@ func hashAlloc(ga *types.GenesisAlloc, isVerkle bool) (common.Hash, error) {
 		emptyRoot = types.EmptyVerkleHash
 	}
 	db := rawdb.NewMemoryDatabase()
-	if isVerkle {
```

Comment:
It's unnecessary to persist any transition flag, as this function is supposed to calculate the state root of genesis only.